libs/membership/website/src/lib/memberplan-picker/memberplan-picker.tsx

```diff
@@ -38,6 +39,8 @@ export const MemberPlanPicker = forwardRef<HTMLButtonElement, BuilderMemberPlanP
 
     const showRadioButtons = memberPlans.length > 1
     const selectedMemberPlan = memberPlans.find(({id}) => id === value)
+    const showPicker =
```

Comment:
Hide picker fully when no image, no description & only 1 memberplan exists.

Have to use toPlaintext as it can have empty children.